annotators/NER/README.md

```diff
@@ -0,0 +1,9 @@
+# Title
+## Description
+
+## Input/Output
+
+**Input**
+**Output**
```

Comment:
?

Reply:
some readmes respawn in their old versions after the branch rebasing(this is a perpetual issue). 
Fixed this one as well.